tests/conftest.py

```diff
@@ -26,6 +25,12 @@ def toloka_url(toloka_api_url) -> str:
     return f'{toloka_api_url}/v1'
 
 
+os.environ['AIRFLOW__CORE__LOAD_DEFAULT_CONNECTIONS'] = 'False'
+os.environ['AIRFLOW__CORE__LOAD_EXAMPLES'] = 'False'
+os.environ['AIRFLOW__CORE__UNIT_TEST_MODE'] = 'True'
+os.environ['AIRFLOW_HOME'] = os.path.dirname(os.path.dirname(__file__))
```

Comment:
Update here would be more readable
```Python
os.environ.update({
    'AIRFLOW__CORE__LOAD_EXAMPLES': 'False',
    'AIRFLOW__CORE__UNIT_TEST_MODE': 'True',
    'AIRFLOW_HOME': os.path.dirname(os.path.dirname(__file__)),
})
```

Still, could you check that this is necessary?